### ISSUE TITLE
Test Coverage for aria attributes on error state inputs

### ISSUE THREAD

**Issue by scottqueen-bixal** [GSA/px-benefit-finder#981]
### Description

Related: #945 

### Acceptance Criteria

- [x] e2e test that expects default `aria-invalid` state for input, select, and radio should be `false`
- [x] e2e test that expects `aria-invalid="true"` when a required field has no value and an error has occurred
- [x] e2e test that expects a select or radio input `aria-invalid="false"` when the previous state was `true` but the error has been resolved
- [x] e2e test that expects a select and  numeric input that composes the date field to have `aria-invalid="false"` when the previous state was `true` but the error on that input in the the date has been resolved
